### DISCUSSION TITLE
Requesting permission to use for a Youtube channel.

### DISCUSSION THREAD

**Discussion by WooDose** [asigalov61/Euterpe-X#1] [General]
Hey! 

First off, amazing work; I've been messing around with the outputs for a few days now and it's not only much more versatile than Musenet since I can access the code directly, but also in general seems to generate more reliably and doesn't go into completely weird directions out of nowhere. I absolutely love the work you've done here and I'm glad there's an amazing replacement for Musenet.

Onto my question; I'm the owner of the youtube channel 43% Artificial. In 2020 I began using musenet and releasing any results I got from it into the channel, mostly to share and to get more people interested in the idea of AI generated music, but after a while I had to stop, and eventually when I wanted to return I found out that Musenet was taken down, so I would like to ask if it's fine for me to use Euterpe-X to start up from where I left, always leaving credit to you within the videos as I did with Musenet. 

Again, as an AI music enthusiast I thank you for your work, and I hope to see this field of work getting a lot more attention in the future!

**Comment by asigalov61** [comment]
@WooDose Thank you for writing and thank you for appreciating my work! It means a lot to me! :)

Yes, you can most certainly use Euterpe-X for any good purpose as it is licensed under Apache 2.0 license. If possible, please distribute your outputs freely as Euterpe-X was trained on publicly available MIDIs which are generally CC-BY-NC-SA licenses.

Otherwise, feel free to use it for your YouTube channel and I would be curious to see the results. :)

MuseNet was indeed a great Symbolic Music AI system and it is indeed a shame that it was depreciated by OpenAI.

If you need assistance or help with the Euterpe-X code/colabs, please let me know. I would be more than happy to help.

Sincerely,

Alex.

**Reply by WooDose** [reply]
Thanks for the reply! I will let you know when I have a few samples made to add them to the samples dir.

**Comment by asigalov61** [comment]
@WooDose PS. If you like, I would be happy to add your output MIDIs generated with Euterpe-X to repo's samples dir as user-generated output with the link and credit to your channel.
